### DISCUSSION TITLE
3.0.5

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-studio#131] [Announcements]
## What's Changed
* Configure Leading edge Debounce Strategy to allow refresh of dropdown at the first render of the activity by @jdevillard in https://github.com/elsa-workflows/elsa-studio/pull/122
* Add `.vs/` folder to gitignore. by @Chryssie in https://github.com/elsa-workflows/elsa-studio/pull/128
* Adding in build steps for NPM by @dalefrancis88 in https://github.com/elsa-workflows/elsa-studio/pull/127
* Allow overriding redirects in WorkflowInstanceEditor and WorkflowInstanceDesigner. by @Chryssie in https://github.com/elsa-workflows/elsa-studio/pull/124
* update login logic to match other API calls by @mohdali in https://github.com/elsa-workflows/elsa-studio/pull/123
* Expose Event + Method to get Selected Workflow Definition by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/130

## New Contributors
* @Chryssie made their first contribution in https://github.com/elsa-workflows/elsa-studio/pull/128
* @dalefrancis88 made their first contribution in https://github.com/elsa-workflows/elsa-studio/pull/127

**Full Changelog**: https://github.com/elsa-workflows/elsa-studio/compare/3.0.4...3.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/elsa-workflows/elsa-studio/releases/tag/3.0.5'>3.0.5</a>.</em>

**Comment by glime-ai** [comment]
Based on the context provided, it appears that there have been several changes and contributions made to the Elsa Workflow Studio project in the transition from version 3.0.4 to 3.0.5. Here's a summary of the changes:

1. A debounce strategy was configured to allow the refresh of dropdown menus at the first render of an activity. This change was made by @jdevillard and can be found in pull request [#122](https://github.com/elsa-workflows/elsa-studio/pull/122).

2. The `.vs/` folder was added to the `.gitignore` file to prevent it from being tracked by Git. This change was made by @Chryssie in pull request [#128](https://github.com/elsa-workflows/elsa-studio/pull/128).

3. Build steps for NPM were added to the project to facilitate the building process. This contribution was made by @dalefrancis88 in pull request [#127](https://github.com/elsa-workflows/elsa-studio/pull/127).

4. Overrides for redirects in the WorkflowInstanceEditor and WorkflowInstanceDesigner were enabled. This change was also made by @Chryssie in pull request [#124](https://github.com/elsa-workflows/elsa-studio/pull/124).

5. The login logic was updated to match other API calls by @mohdali in pull request [#123](https://github.com/elsa-workflows/elsa-studio/pull/123).

6. An event and method to get the selected Workflow Definition were exposed. This was done by @sfmskywalker in pull request [#130](https://github.com/elsa-workflows/elsa-studio/pull/130).

Additionally, two new contributors, @Chryssie and @dalefrancis88, made their first contributions in this release cycle.

For the full list of changes, you can view the full changelog by visiting the comparison between versions 3.0.4 and 3.0.5 on the GitHub repository: [Full Changelog](https://github.com/elsa-workflows/elsa-studio/compare/3.0.4...3.0.5).
